CHANGELOG.asciidoc

```diff
@@ -11,15 +11,15 @@ https://github.com/elastic/beats/compare/v7.13.1...v7.13.2[View commits]
 
 *Affecting all Beats*
 
-- Fix ILM alias creation when write alias exists and initial index does not exist {pull}26143[26143]
-- Add FAQ entry for madvdontneed variable {pull}23429[23429]
+- Fix ILM alias creation when write alias exists and initial index does not exist. {pull}26146[26146]
+- Add FAQ entry for madvdontneed variable. {pull}23429[23429]
```

Comment:
@andresrc Not sure we should include this since it's a doc change and only seems to apply to beats prior to 7.10.2. WDYT?

Reply:
Let's remove it, yes, thanks!